simapp/codec/codec.proto

```diff
@@ -56,3 +60,38 @@ message MsgSubmitEvidence {
   Evidence                                       evidence = 1;
   cosmos_sdk.x.evidence.v1.MsgSubmitEvidenceBase base     = 2 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
+
+// MsgSubmitProposal defines the application-level message type for handling
+// governance proposals.
+message MsgSubmitProposal {
+  option (gogoproto.equal)           = true;
+  option (gogoproto.goproto_getters) = false;
+
+  Content                                   content = 1;
+  cosmos_sdk.x.gov.v1.MsgSubmitProposalBase base    = 2 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
```

Comment:
We should be consistent with either `base` or `content` first between this and `Proposal`. I suggest `base` first.

Reply:
Good catch!